docs/usage/configuration-options.md

```diff
@@ -525,6 +525,16 @@ If you need to _override_ constraints that Renovate detects from the repository,
 !!! note
     Make sure not to mix this up with the term `compatibility`, which Renovate uses in the context of version releases, e.g. if a Docker image is `node:12.16.0-alpine` then the `-alpine` suffix represents `compatibility`.
 
+## constraintsFiltering
+
+This option allows you to select the type of release filtering you want. Currently we have these two options:
+`none`: No release filtering
+`strict`: If any of the release's constraints match, then it's included
+
+We are working on adding more advanced filtering options.
+
+Note: There must be a `constraints` object present in your renovate config for this to work.
+
```

Comment:
This feature is limited to `pypi` datasource only.

Reply:
Should I use admonitions here?

!!! note 
- <listItem 1>
- <listItem 2>

![image](https://user-images.githubusercontent.com/76607510/214229769-bce3b609-5fab-44d3-83bc-dce67affd3df.png)

Reply:
I am not sure if this list supported. I just made this using inspect element to show an example :)
Also there are many cases where we don't us admonitions for `Note:`, is htere a reason or they are pending?